xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
 
+<a name="Introduction"/>
```

Comment:
Take care changing anchors. It looks like you didn't change this in the TOC. Anchors should be all lower case and 'introduction' is not a good anchor as there may be multiple introductions across the documents/sections. Maybe ```partone-introduction```.